src/app/views/provisioning/subscriptions.html

```diff
@@ -99,7 +99,15 @@ <h2 class="section-title" ng-hide="vm.cartSubscriptions" translate>
           <span class="cell-label visible-xs-inline">
             {{ 'mno_enterprise.templates.dashboard.provisioning.subscriptions.licenses' | translate }}:
           </span>
-          <span title="{{subscription.max_licenses}}">{{subscription.max_licenses}}</span>
+          <div ng-show="vm.subscription.max_licenses">
+            {{ vm.subscription.max_licenses }}
+          </div>
+          <div ng-hide="vm.subscription.max_licenses">
+            {{ 'mno_enterprise.templates.dashboard.provisioning.confirm.no_licenses.message' | translate }}
```

Comment:
This shouldn't use this locale namespace "mno_enterprise.templates.dashboard.provisioning.confirm" but "mno_enterprise.templates.dashboard.provisioning.subscriptions".
Even if that create some duplication in the locales, this allows us to precisely control the text.